layouts/shortcodes/katacoda-removal.html

```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-secondary callout note" role="alert">
+  <strong>{{ T "note" | safeHTML }}</strong> {{ T "katacoda_message" | safeHTML }}
+</div>
```

Comment:
I'd use a pageinfo:
```suggestion
<div class="pageinfo pageinfo-secondary">
{{ T "katacoda_message" | safeHTML }}
</div>
```

Reply:
Agreed, I've made that update in a follow up PR https://github.com/kubernetes/website/pull/40304